### PR TITLE
Bump PROW GKE Staging images to v0.3.1

### PR DESCRIPTION
**Pull request by davidz627**
/assign @msau42 
/cc @verult

### CODE REVIEW

**Comment by davidz627** [comment]
/hold
until image `gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver:v0.3.1-gke.0` exists

File: deploy/kubernetes/overlays/prow-gke-staging/controller_images.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ spec:
         - name: csi-attacher
           image: gcr.io/gke-release-staging/csi-attacher:v1.0.1-rc1
```
Comment (msau42) [inline deploy/kubernetes/overlays/prow-gke-staging/controller_images.yaml]: we should bump the sidecars too

Comment (davidz627) [reply deploy/kubernetes/overlays/prow-gke-staging/controller_images.yaml]: done

**Comment by davidz627** [comment]
/hold cancel
tags exist in staging as of `December 17, 2018 at 4:05:44 PM UTC-8`

**Pull request merged**